### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by max8899**
taht -> that

### CODE REVIEW

File: pkg/framework/path/selectors/interface.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type Interface interface {
 	// AsSlice returns a selector that selects Slices from the given
 	// values.
 	AsSlice() Slice
-	// Number returns a selector taht selects Numbers from the given values.
+	// Number returns a selector that selects Numbers from the given values.
```
Comment (nikhita) [inline pkg/framework/path/selectors/interface.go]: Thanks for catching this, @max8899! :+1:

**Comment by nikhita** [comment]
/assign @soltysh 
for approval

**Comment by nikhita** [comment]
@max8899 congratulations on your first PR to kubectl! :tada:

**Comment by max8899** [comment]
@nikhita thank you very much and looking forward to make more contributions in the future.

**Comment by adohe** [comment]
@max8899 Thanks very much :)

**Pull request merged**